roblox/assets.py

```diff
@@ -119,27 +119,27 @@ class EconomyAsset(BaseAsset):
     It is intended to parse data from https://economy.roblox.com/v2/assets/ASSETID/details.
 
     Attributes:
-        id: Id of the Asset
-        product_id: Product id of the asset
-        name: Name of the Asset
-        description: Description of the Asset
-        type: Type of the Asset
```

Comment:
Maybe we shouldn't need to write "... of the Asset." over and over, although I don't know what exactly to put there instead

Reply:
Could make it "The asset's ..." but I'm not really sure if that's better.

Reply:
I have updated the attribute descriptions to have less repetition is it better now?